package.json

```diff
@@ -22,5 +22,8 @@
   "homepage": "https://github.com/kamp-us/monorepo#readme",
   "devDependencies": {
     "turbo": "^1.6.3"
+  },
+  "volta": {
+    "node": "19.2.0"
```

Comment:
wouldn't it be better to make this the version of LTS?

Reply:
I think it would better. Thank you for the suggestion. I checked it it looks like `18.14.12` has LTS. I'm updating my pr 🙏

Reply:
Pr has updated 💃